PokeAlarm/Manager.py

```diff
@@ -245,7 +246,23 @@ def handle_pokemon(self, pkmn):
         height, weight, gender = pkmn['height'], pkmn['weight'], pkmn['gender']
         if gender != '?':
             gender = u'\u2642' if gender is 1 else u'\u2640' if gender is 2 else u'\u26b2' # male, female, neutral
-        
+
+        # Check the sizes of the Pokemon
+        size = None
```

Comment:
Size should really default to 'unknown' (because they may still attempt to use size as a DTS